qdrant-landing/content/documentation/tutorials/async-api.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Asynchronous API
+weight: 18
+---
+
+# Using Qdrant asynchronously
+
+Databases are often launched as separate services and are accessed via a network. All the interactions with them are IO-bound and can 
+be performed asynchronously so as not to waste time actively waiting for a server response. If you use Python, that is achieved by 
+using `async/await` syntax. That lets the interpreter switch to another task while waiting for a response from the server.
+
+Qdrant exposes two interfaces: HTTP and gRPC. The official SDKs for different languages are based on the autogenerated clients, and
+in Python [qdrant-client](https://github.com/qdrant/qdrant-client), you can call all the methods asynchronously. This tutorial presents 
+how to do it.
+
+## When the usage of async API is justified?
+
+If the application you are writing will never support multiple users at once, for example, it is a script run once daily, then there 
+is no need to use async API. But if you are writing a web service that multiple users will use simultaneously, you shouldn't be 
+blocking the threads of the web server as it limits the number of concurrent requests it can handle. In this case, you should use 
+the async API.
+
+Modern web frameworks like [FastAPI](https://fastapi.tiangolo.com/) and [Quart](https://quart.palletsprojects.com/en/latest/) support 
+async API out of the box. Asynchronous code cannot be incorporated into an existing synchronous codebase, so it's better to choose 
+the proper tool from the start.
+
+## How to use async API?
+
+As usual, calling any method of Qdrant requires establishing a connection to the server. We need to create an instance of `QdrantClient`
+that will act as a gateway. If you want to do it locally, please make sure Qdrant server is running. If it's not, then you can launch it
+in a Docker container:
+
+```bash
+docker run -p "6333:6333" -p "6334:6334" qdrant/qdrant:v1.4.0
+```
+
+Alternatively, you can also use a [Qdrant Cloud](https://cloud.qdrant.io/) cluster. In this case, you need to pass the API key to the
+client constructor. No matter which way you choose, the `QdrantClient` instance has to be created:
+
+```python
+from qdrant_client import QdrantClient
+
+client = QdrantClient("localhost")
+# If you use Qdrant Cloud, then pass the API key to the constructor:
+# client = QdrantClient("https://your-cluster-url.cloud.com", api_key="your-api-key")
+```
+
+The top-level client exposes just the synchronous methods. We can access the autogenerated async clients and call the underlying methods.
+It is possible to use asynchronous HTTP API by using `qdrant_client.http.api_client.AsyncApiClient`. However, the gRPC API is much more 
```

Comment:
Please put an example here on how you would call it given the code you gave above. I think it would be 

For example using the client we created above `client.http.api_client.AsyncApiClient.<method>`.

Reply:
Our Python SDK does not expose the async HTTP client that easily.